Nodejs/Product/Nodejs/ProjectTemplates/TypeScriptAzureNodejsWorker/Worker.njsproj

```diff
@@ -34,13 +31,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <TypeScriptCompile Include="server.ts" />
-    <TypeScriptCompile Include="startup.ts" />
+    <Compile Include="server.ts" />
```

Comment:
The "Compile" action gets ignores by `tsc` as it's now configured by `tsconfig.json`. Regardless, it's still needed in order for VS to keep track of `dirty` files and to trigger the build if necessary.

I have not found a way to avoid this.

Reply:
This is no longer the case. That logic got removed and now the ts file are always "dirty". The TS SDK and tsc are now able to implement its own incremental build.